src/Rendering.hs

```diff
@@ -0,0 +1,37 @@
+module Rendering where
+
+import Graphics.Gloss
+
+import Game
+import Data.Array
+
+boardGridColor = makeColorI 255 255 255 255 
```

Comment:
Change the color of the grid because it is not visible since the background is the same color.